eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/trace/Trace.java

```diff
@@ -111,7 +112,17 @@ public Span addTraceInfoToSpan(ChannelHandlerContext ctx, CloudEvent cloudEvent)
         }
         Context context = ctx.channel().attr(AttributeKeys.SERVER_CONTEXT).get();
         Span span = context != null ? context.get(SpanKey.SERVER_KEY) : null;
-        return addTraceInfoToSpan(span, cloudEvent);
+
+        if (span == null) {
+            log.warn("span is null when finishSpan");
+            return null;
+        }
+
+        //add trace info
+        for (String entry : cloudEvent.getExtensionNames()) {
+            span.setAttribute(entry, cloudEvent.getExtension(entry) == null ? "" : Objects.requireNonNull(cloudEvent.getExtension(entry)).toString());
+        }
+        return span;
```

Comment:
Why do we need to inline the `addTraceInfoToSpan` method here? I think it is enough to add a `requireNonNull` method in the "add trace info" step of `addTraceInfoToSpan` method.

Reply:
> I think it is enough to add a `requireNonNull` method in the "add trace info" step of `addTraceInfoToSpan` method.

I found that your current suggestion contradicts your previous one (https://github.com/apache/eventmesh/pull/3632#issuecomment-1776816518). So, why you now think it's needed to add an extra `requireNonNull` to `addTraceInfoToSpan` method?

Reply:
Excuse me for overlooking some folded context earlier. I initially intended to point out that the `addTraceInfoToSpan` method should not be inlined here. However, it appears that further review is unnecessary as the issue corresponding to this PR is invalid. I will close the issue and label this PR as `invalid`.

@harshithasudhakar Thank you very much for your contribution. Please feel free to select other valuable issues to work on.